Update beta-testing.md (#258)
AdguardTeam/KnowledgeBaseVPN: docs/miscellaneous/beta-testing.md

Once on the beta testing page, all you have to do is choose between the beta and the nightly versions, download the respective build and fill in the [form](https://surveys.adguard.com/en/vpn_beta_testing_program/form.html) to apply.

::: note

To get the AdGuard VPN for iOS beta or nightly build, first you have to install [Testflight](https://apps.apple.com/app/testflight/id899247664) on your iOS device.

:::

### How to report a bug

Let’s imagine it happened — you spotted a bug. Or maybe you just want to share your suggestion with developers? Let us know: go to [GitHub](https://github.com/AdguardTeam/), find a suitable repository, create a new issue there, and describe the bug/feature request in detail.
If you spotted a bug on the build you are testing or just want to share your suggestion with our developers, let us know: go to [GitHub](https://github.com/AdguardTeam/), find a suitable repository, create a new issue there, and describe the bug/feature request in detail.

> The above method is the most preferred, but you can use others if necessary:

Author: мне кажется, most preferred - это плеоназм

> - Find us on [social media](https://adguard-vpn.com/en/discuss.html). This is not the most efficient way to discuss technical matters, but we'll aways try to be helpful.

Reviewer: ```suggestion
- Find us on [social media](https://adguard-vpn.com/discuss.html). This is not the most efficient way to discuss technical matters, but we’ll aways try to be helpful.
```

### How to become a beta or nightly tester

Our requirements are easy: use AdGuard, upgrade to the beta/nightly version when it becomes available, and report any bugs you notice. You will need a [GitHub](https://github.com/) account to become our tester, so you may want to get that out of the way first.

> To become an AdGuard VPN tester, you need the device on which you will install the app and [the link to the the beta testing page](https://adguard-vpn.com/beta.html). There you will find more information on how to join our program and test AdGuard VPN for Windows, Mac, Android, iOS, Linux, and AdGuard VPN Browser Extension.

Reviewer: ```suggestion
You will also need the device on which you will install the app and [the link to the the beta testing page](https://adguard-vpn.com/beta.html). There you will find more information on how to join our program and test AdGuard VPN for Windows, Mac, Android, iOS, Linux, and AdGuard VPN Browser Extension.
```
